src/index.js

```diff
@@ -1,33 +1,23 @@
 import axios from "axios"
 import babelPolyfill from "babel-polyfill"
 import humps from "humps"
-
-class KisiError extends Error {
-    constructor(status, code = "000000", reason = null) {
-        super("An error occurred interacting with the Kisi API.")
-
-        this.status = status
-        this.code = code
-        this.reason = reason
-    }
-}
+import handleError from "./errors"
```

Comment:
Would sort this lexically just for consistency reasons :) or maybe there is a system?